ReactiveObjC.xcodeproj/project.pbxproj

```diff
@@ -2579,13 +2600,19 @@
 					"DTRACE_PROBES_DISABLED=1",
 				);
 				INFOPLIST_FILE = ReactiveObjC/Info.plist;
+				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/../Frameworks @loader_path/Frameworks";
```

Comment:
This shouldn't be needed, I don't think. It should already be specified in the `xcconfig`?

Reply:
Right, this was probably an automatic Xcode change when adding the first Swift code to the target.

Reply:
I removed all the redundant `LD_RUNPATH_SEARCH_PATHS` settings in commit 31e9f4aca5705c0f1f05df1f3340a4a869b2dbcb.